src/components/Message/Message.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+type Props = {};
+
+type State = {
+  query: string;
+};
+
+export class Message extends React.Component<Props, State> {
+  state: State = {
+    query: '',
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ query: event.key });
+    });
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ query: event.key });
+    });
```

Comment:
it's two different functions here, so remove event listener will not work as you can expect, you should write that func in a separate function